modules/drivers/bigquery-cloud-sdk/test/metabase/test/data/bigquery_cloud_sdk.clj

```diff
@@ -116,7 +116,7 @@
 
 ;; Fields must contain only letters, numbers, and underscores, start with a letter or underscore, and be at most 128
 ;; characters long.
-(def ^:private ValidFieldName #"^[A-Za-z_]\w{0,127}$")
+(def ^:private ValidFieldName #"^[A-Za-z_][\w-]{0,127}$")
```

Comment:
This doesn't match the description above - I suppose dashes are actually legal in BigQuery?

Reply:
Thanks, good spotting - meant to revert that after proving it wasn't accepted by BigQuery.

Fixed in the origin pull request in 10e13647b1.

Reply:
Pulled from the upstream and updated this branch.